install/prepare_site_init.md

```diff
@@ -452,7 +453,28 @@ the system size.
         "$min_amount"
     done
     ```
+### Enable Rack Resiliency
 
+This section can be skipped if Rack Resiliency feature is not required.
+
+1. (`pit#`) Update the customizations file.
+
+    ```bash
+      yq write -i ${SITE_INIT}/customizations.yaml \
+        'spec.kubernetes.services.rack-resiliency.enabled' "true"
+    ```
+1. (`pit#`) If site specific prefix for Kubernetes and `Ceph` zones is required:
+
+   For example:
+   
+   ```bash
+      yq write -i ${SITE_INIT}/customizations.yaml \
+        'spec.kubernetes.services.rack-resiliency.k8s_zone_prefix' "prefix-string"
```

Comment:
This example makes it seem like the prefix for both is expected to match. Is that a requirement?